adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -867,20 +1314,41 @@ abstract class AvroGenomicRDD[T <% IndexedRecord: Manifest, U <: AvroGenomicRDD[
     with GenomicRDD[T, U] {
 
   /**
+   * This method is where we save the partition map to the disk. This is done
```

Comment:
`This method is where we save the partition map to the disk.` &rarr; `Save the partition map to the disk.`